### PR TITLE
Add support of deserialization of unbounded datetime intervals

### PR DESCRIPTION
**Pull request by manifest**

### CODE REVIEW

File: src/serde.rs [hunk 1]
```diff
@@ -141,83 +145,3 @@ pub(crate) mod ts_seconds_option_bound_tuple {
     }
 }
 
-////////////////////////////////////////////////////////////////////////////////
-
-#[cfg(test)]
```
Comment (0nkery) [inline src/serde.rs]: Why did you remove tests?

Comment (manifest) [reply src/serde.rs]: Haven't had time to write proper test.

Comment (0nkery) [reply src/serde.rs]: I think that old tests should be left as-is (or slightly changed) even if you haven't had time to write new ones.

Comment (manifest) [reply src/serde.rs]: I've got it back
